pkg/vpcmodel/output.go

```diff
@@ -261,14 +270,18 @@ func (of *serialOutputFormatter) AggregateVPCsOutput(outputList []*SingleAnalysi
 		// plain concatenation
 		vpcsOut := make([]string, len(outputList))
 		hasStatelessConn := false
+		hasOverApproximatedConn := false
```

Comment:
Please consider encapsulating `hasStatelessConn` and `hasOverApproximatedConn` into a struct

Reply:
this messaging mechanism is about to be rewritten. am not going to do it in this PR